beamer/tests/agent/unit/test_invalidate.py

```diff
@@ -159,3 +163,39 @@ def test_maybe_invalidate_claim_wrong_fill_id_but_timed_out(mocked_invalidate):
     process_claims(context)
     assert claim.is_claimer_winning
     assert mocked_invalidate.called
+
+
+@patch("beamer.chain._withdraw")
+@pytest.mark.parametrize(
+    "test_data",
+    [
+        (ACCOUNT.address, ADDRESS1, False),
+        (ADDRESS1, ACCOUNT.address, True),
+        (ADDRESS1, make_address(), False),
+    ],
+)
+def test_maybe_withdraw_after_invalidation(mocked_withdraw, test_data):
+    context, _ = make_context()
+
+    request = make_request()
+    context.requests.add(request.id, request)
+
+    participant1 = test_data[0]
+    participant2 = test_data[1]
```

Comment:
renaming it to claimer and challenger would be more explicit

Reply:
✔️